aptos-move/framework/aptos-stdlib/doc/bls12381.md

```diff
@@ -33,9 +33,13 @@ as per the [IETF BLS draft standard](https://datatracker.ietf.org/doc/html/draft
 -  [Function `aggr_or_multi_signature_from_bytes`](#0x1_bls12381_aggr_or_multi_signature_from_bytes)
```

Comment:
i don't know where most of the change in this file came from...

Reply:
Uhm... which ones are you unsure about?